packages/next/src/cli/next-dev.ts

```diff
@@ -333,16 +343,4 @@ const nextDev: CliCommand = async (args) => {
   await runDevServer(false)
 }
 
-function cleanup() {
-  if (!child) {
-    return
-  }
-
-  child.kill('SIGTERM')
-}
-
-process.on('exit', cleanup)
-process.on('SIGINT', cleanup)
-process.on('SIGTERM', cleanup)
```

Comment:
the other handlers were already killing the child - this one always sends SIGTERM. However, we still need to add a special handler for the exit event since it needs to be synchronous